tests/substrate_tests/builtins.rs

```diff
@@ -491,7 +491,7 @@ fn addmod() {
         r##"
         contract x {
             function test() public {
-                assert(addmod(500, 100, 0) == 200);
+                assert(addmod(500, 100, 0) == 600);
```

Comment:
When I run this in remix IDE I get `0`, maybe we should do that too instead? (I have no hard opinion here)

Reply:
This function fails. It does not return anything. Instead, an error is thrown by the environment.

Reply:
It returns zero now, just like solc.